app/views/homepage/_promotion-slots.html.erb

```diff
@@ -2,119 +2,55 @@
   Before updating a promotion slot please read the documentation.
   See: https://docs.publishing.service.gov.uk/apps/frontend/update-homepage-promotion-slots.html
 %>
-<section>
+<section class="homepage-section">
+  <div class="homepage-section__heading">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t('homepage.index.featured')
+    } %>
+  </div>
+
   <div class="govuk-grid-row" data-module="gem-track-click">
+    <% t("homepage.index.promotion_slots").each do | item | %>
     <div class="govuk-grid-column-one-third">
-      <a href="/government/topical-events/cop26"
-         aria-hidden="true"
-         tabindex="-1"
-         data-track-category="homepageClicked"
-         data-track-action="promoImageLink"
-         data-track-label="/government/topical-events/cop26"
-         data-track-value="1"
-         data-track-dimension-index="29"
-         data-track-dimension="homepage/cop26.jpg">
-        <%= image_tag "homepage/cop26.jpg", {
-          alt: "",
-          class: "home-promo__image",
-          height: 407,
-          loading: "lazy",
-          width: 610,
-          sizes: "(max-width: 640px) 100vw, (max-width: 1020px) 33vw, 300px",
-          srcset: {
-            "homepage/cop26.jpg": "610w",
-            "homepage/cop26-480.jpg": "480w",
-            "homepage/cop26-320.jpg": "320w",
-            "homepage/cop26-240.jpg": "240w",
-            "homepage/cop26-170.jpg": "170w",
-          },
-        } %>
-      </a>
-      <h3 class="home-promo__heading">
-        <a href="/government/topical-events/cop26"
-           class="govuk-link home-promo__link"
-           data-track-category="homepageClicked"
-           data-track-action="promoTextLink"
-           data-track-label="/government/topical-events/cop26"
-           data-track-value="1"
-           data-track-dimension-index="29"
-           data-track-dimension="<%= t('homepage.index.promo_boxes.title_1') %>">
-          <%= t('homepage.index.promo_boxes.title_1') %>
+      <div class="home-promo">
+        <a href="<%= item[:href] %>"
+          aria-hidden="true"
+          tabindex="-1"
+          data-track-category="homepageClicked"
+          data-track-action="promoImageLink"
+          data-track-label="<%= item[:href] %>"
+          data-track-value="1"
+          data-track-dimension-index="29"
+          data-track-dimension="<%= item[:image_src] %>">
+          <%= image_tag item[:image_src], {
+            alt: "",
+            class: "home-promo__image",
+            height: 407,
+            loading: "lazy",
+            width: 610,
+            sizes: "(max-width: 640px) 100vw, (max-width: 1020px) 33vw, 300px",
+            srcset: item[:srcset]
```

Comment:
[Multi-line hashes should have a trailing comma][1] for a better diff with future changes:

```suggestion
            srcset: item[:srcset],
```

[1]: https://github.com/alphagov/rubocop-govuk/blob/ce655779e45892db6ac00d6789c652ec2e506748/config/style.yml#L23-L28